src/components/PaginationFooter/PaginationContent.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { memo } from 'react';
+import type { PaginationContentProps } from './types';
+import PaginationProvider from './context/PaginationProvider';
+import DisplayedRowsLabel from './DisplayedRowsLabel';
+import RowsPerPage from './Dropdowns/RowsPerPage';
+import SelectPage from './Dropdowns/SelectPage';
+import FirstPage from './Buttons/FirstPage';
+import PreviousPage from './Buttons/PreviousPage';
+import NextPage from './Buttons/NextPage';
+import LastPage from './Buttons/LastPage';
+
+const PaginationContent = (props: PaginationContentProps) => (
```

Comment:
This component have been refactored into multiple smaller components. Where props are provider using context.